var/spack/repos/builtin/packages/py-cppheaderparser/package.py

```diff
@@ -0,0 +1,18 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class PyCppheaderparser(PythonPackage):
+    """Parse C++ header files and generate a data structure
+    representing the class"""
+
+    homepage = "https://pypi.org/project/CppHeaderParser"
+    url      = "https://pypi.io/packages/source/C/CppHeaderParser/CppHeaderParser-2.7.4.tar.gz"
+
+    version('2.7.4', sha256='382b30416d95b0a5e8502b214810dcac2a56432917e2651447d3abe253e3cc42')
+
+    depends_on('py-setuptools', type='build')
```

Comment:
Missing build/run dep on `py-ply`

Reply:
Do you want me to add new dependency with py-ply??. It built find without this dependency.
depends_on('py-ply', type=('build', 'run'))